tailwind.config.js

```diff
@@ -52,6 +52,17 @@ module.exports = {
         small: "0 5px 10px rgba(0, 0, 0, 0.12)",
         medium: "0 8px 30px rgba(0, 0, 0, 0.12)",
       },
+      fontFamily: {
+        "belly-display": "Bely Display",
+        "public-sans": "Public Sans",
```

Comment:
Let's define some fallback fonts:

```suggestion
        "belly-display": "'Bely Display', serif",
        "public-sans": "'Public Sans', sans-serif",
```